webhooks/webhooks_test.go

```diff
@@ -655,6 +655,7 @@ func TestParse_FailedPaymentNotification(t *testing.T) {
 		Transaction: webhooks.Transaction{
 			XMLName:          xml.Name{Local: "transaction"},
 			UUID:             "a5143c1d3a6f4a8287d0e2cc1d4c0427",
+			InvoiceID:        "8fjk3sd7j90s0789dsf099798jkliy65",
```

Comment:
I get that this is just dummy data, but it raised an issue for me: It is confusing that an invoiceID on a Transaction [here](https://recurly.com/developers/api/v2021-02-25/index.html#operation/list_account_transactions) looks like a string of <= 13 characters, and on the webhook response [here](https://recurly.com/developers/api/v2021-02-25/index.html#operation/list_account_transactions), it looks longer. At first I thought maybe it was a UUID being called an ID, but the example is not a valid v4 UUID. Kind of a continuation of the confusion you brought up this morning.

Reply:
I don't really have a point... It's just annoying. I'll note that in v2, it was a URL _to_ an invoice.

Reply:
Yeah, I just copied what was in the fixture, which is this random string. As consumers, we do not own the formatting requirements of this field's data, so it would be inappropriate for us to attempt to validate or test it to a specific format that Recurly is free to change at any time. That is: our code should not need to know the format requirements on Recurly's side - just use whatever they give us.

It's probably partially due to that v2 URL -> v3 UUID that this test is just a random string.